opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -337,6 +346,23 @@ enum DataPointFlags {
   // Bits 2-31 are reserved for future use.
 }
 
+// GaugeType defines valid gauge "sub-types", which are used to describe
+// gauges which have additional properties. These sub-types generally apply
+// to metrics which originate from protocols which support these additional
+// types, but are represented as gauges in OTLP.
+enum GaugeType {
+  // GAUGE_TYPE_STANDARD is the default GaugeType, which represents a gauge
+  // that does not have a sub_type. Since the OpenTelemetry API enforces types,
+  // this subtype is used for all metrics from OpenTelemetry instrumentation.
+  GAUGE_TYPE_STANDARD = 0;
+
+  // GAUGE_TYPE_UNKNOWN is a GaugeType which represents a metric for
+  // which the type is not known, such as a Prometheus metric without a TYPE
+  // comment. For example, the metric may be a monotonically-increasing counter,
+  // or histogram, but type information was not included at the source.
+  GAUGE_TYPE_UNKNOWN = 1;
```

Comment:
// GAUGE_TYPE_UNKNOWN is a GaugeType which represents a metric for
  // which the real type is unknown is treated as a Gauge, such as a Prometheus
  // metric without a TYPE comment. For example, the metric may be a
  // monotonically-increasing sum, or histogram bucket counter, but type information
  // was not included at the source.

Reply:
done